gen3-integration-tests/pages/login.py

```diff
@@ -112,8 +113,10 @@ def ras_login(self, page: Page):
         page.locator(self.RAS_USERNAME_INPUT).fill(os.environ["CI_TEST_RAS_USERID"])
         page.locator(self.RAS_PASSWORD_INPUT).fill(os.environ["CI_TEST_RAS_PASSWORD"])
         ras_login_button.click()
+        time.sleep(3)
```

Comment:
can we try and see if this eliminates the need for hard wait - https://playwright.dev/python/docs/api/class-locatorassertions#locator-assertions-to-be-enabled

Reply:
I dont think its the problem of button being enabled. The playwright was proceeding without even going into the if statement. I did try to increase the timeout but that didnt help. Tried the below code as well:
if locator.is_enabled():

Finally when time.sleep was used, it went inside the if statement and clicked on the button

Reply:
hmm, I can't see how a timeout solves that

Reply:
I understand, even I am not quite sure why the timeout didnt work and time.sleep did work but this solution works. I can try exploring more on it to see